release-notes.txt

```diff
@@ -1,3 +1,619 @@
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release
+==================
+
+The 2023.04 release includes:
+
+- Mass Storage Class Support in USBUS
+- Initial support for libSCHC
+- FlashDB support
+- BLE downlink in GNRC Border Router
+- New boards:
+  - esp32c3-wemos-mini
+  - esp32s2-lilygo-ttgo-t8
+  - esp32s2-wemos-mini
+  - esp32s3-pros3
+  - rpi-pico-w
+  - sipeed-longan-nano
+- New device drivers:
+  - Black/white e-Paper/e-Ink display
+  - RFID reader (MFRC522)
+  - ST VL53L1X Time-of-Flight sensor
+
+280 pull requests, composed of 595 commits, have been merged since the
+last release, and 3 issues have been solved. 27 people contributed with
+code in 83 days. 1246 files have been touched with 177090 (+) insertions and
+36429 deletions (-).
+
+
+Notations used below
+====================
+
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+
+New features and changes
+========================
+
+Core
+----
+
++ core/compiler_hints: add likely() / unlikely() hints (#19156)
+* core/xfa: disable asan on llvm (#19271)
+* pkg/libfixmath: bump version (#19182)
+
+System Libraries
+----------------
+
++ drivers/wdt: add periph_wdt_auto_start for early watchdog (#18257)
++ fuzzing: Add uri_parser setup (#19057)
++ Revert "sys/pm_layered: pm_(un)block add attribute optimize(3)" (#19155)
++ sys/coding: add XOR based coding module (#17045)
++ sys/event: add event sources (#18758)
++ sys/irq: Add C++ wrapper using RAII (#17066)
++ sys/phydat: add functions for Unix time conversion to phydat (#19080)
++ sys/trickle: cleanup deps (#19404)
++ sys/vfs: add force option to vfs_umount() (#19320)
++ usbus/msc: add CONFIG_USBUS_MSC_AUTO_MTD option to create LUNs on
+  init (#19356)
++ usbus/msc: add initial Mass Storage Class support (#19242)
++ USBUS: Add URB support (#17091)
++ ztimer: add ztimer_stopwatch convenience functions (#19343)
+* bootloaders: fix bootloader button logic (#19365)
+* makefiles/suit: make it possible to accept multiple SUIT keys (#18459)
+* sys/bitfield: don't touch unrelated bits in bf_{set, clear}_all() (#19400)
+* sys/cpp11-compat: Remove xtimer deps (#19369)
+* sys/fmt: optimize scn_u32_dec scn_u32_hex (#19027)
+* sys/fmt: use `fflush(); stdio_write()` instead of `fwrite()` (#19250)
+* sys/shell_lock: do not call strlen, less jumpy (#19157)
+* sys/tiny_strerror: make use of flash_utils.h (#19332)
+* test_utils/print_stack_usage: use thread.h API (#19226)
+* usbus/dfu: do not create alt interface if NUM_SLOTS=1 (#19386)
+
+Networking
+----------
+
++ gcoap/fileserver: add event callbacks (#18414)
++ gnrc_sixlowpan_frag_sfr_congure: add congure_abe support (#16171)
++ gnrc_sixlowpan_frag_sfr_congure: add congure_quic support (#16159)
++ gnrc_sixlowpan_frag_sfr_congure: add congure_reno support (#16170)
++ gnrc_sixlowpan_frag_sfr_congure_sfr: initial import (#16158)
```

Comment:
These can be summarized as: `gnrc_sixlowpan_frag_sfr: optional congestion control support`.

Reply:
done!